config/core-site.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+
+<!-- WASB (Azure Storage) Configuration  -->
+<!-- <property>
+  <name>fs.AbstractFileSystem.wasb.Impl</name>
+  <value>org.apache.hadoop.fs.azure.Wasb</value>
+</property>
+<property>
+  <name>fs.azure.account.key.$storage_account_name.blob.$storage_account_suffix</name>
+  <value>$storage_account_key</value>
+</property> -->
+
+<!-- ADL (Azure Data Lake) Configuration -->
```

Comment:
link to somewhere in docs on how to get keys?

Reply:
Yeah.. this is inherently difficult... You have to link the ADL with your AAD application... I had to piece that together myself from lots of different blogs. Probably worth writing on our docs on that. I'll leave that as a separate item though, since all the users who want this today already know how to do that.